src/main/java/io/odpf/firehose/config/KafkaConsumerConfig.java

```diff
@@ -48,4 +48,8 @@ public interface KafkaConsumerConfig extends AppConfig {
     @ConverterClass(ConsumerModeConverter.class)
     @DefaultValue("SYNC")
     KafkaConsumerMode getSourceKafkaConsumerMode();
+
+    @Key("SOURCE_KAFKA_CONSUMER_COMMIT_DELAY_MS")
```

Comment:
Also, even though negative value doesn't impact the flow. It would be good if we don't accept negative values. Thoughts?

Reply:
Doesn't `SOURCE_KAFKA_CONSUMER_MIN_COMMIT_INTERVAL_MS` make more sense as a config value rather than saying delay. Since we aren't delaying the commit, we are ensuring the minimum interval between the commits ?

Reply:
@lavkesh So what is final decision on config name `interval` or `delay` ?

Reply:
I am thinking `SOURCE_KAFKA_CONSUMER_CONFIG_AUTO_COMMIT_MIN_DELAY_MS`

Reply:
Agree, interval makes more sense. `SOURCE_KAFKA_CONSUMER_CONFIG_MANUAL_COMMIT_MIN_INTERVAL_MS` sounds good